coralproject/talk
# [CORL-1357] Intersection Provider Fixes

## What does this PR do?

Fixes race condition where callbacks were not checked before execution. This also replaces usage of the `withInView` with `useInView`.



## What changes to the GraphQL/Database Schema does this PR introduce?

None.



## How do I test this PR?



Load the comment stream and do not notice errors relating to intersection observers. Try to scroll on admin where you'd expect infinite scroll to occur, notice it working!
